### ISSUE TITLE
do i need to select nodejs 10.x while uploading lambdafunction.zip to aws account?

### ISSUE THREAD

**Issue by praveenbandi86** [slippens/MicroStrategy#1]
what is the runtime for lambdafunction.zip?

**Comment by slippens** [comment]
Yes, the sample was written using a now deprecated version of nodejs, so you should select the latest. I do not believe there should be any major changes from the node version - however, I believe they also removed the ability to upload a lambda function via ZIP file through the GUI so you also many need to use CLI to deploy the code with its required frameworks to lambda.
